### PR TITLE
Tilde fix

### PR DESCRIPTION
**Pull request by btecu**

### CODE REVIEW

File: lib/rules/lint-block-indentation.js [hunk 1]
```diff
@@ -224,6 +224,8 @@ module.exports = function(addonContext) {
         } else {
           childStartColumn = firstNonWhitespace;
         }
+      } else if (this.sourceForNode(child) === '') {
```
Comment (rwjblue) [inline lib/rules/lint-block-indentation.js]: What type of node is `child` in this case?

Comment (btecu) [inline lib/rules/lint-block-indentation.js]: `ElementNode`

**Comment by kmiyashiro** [comment]
Are there any blockers for this fix?

**Comment by rwjblue** [comment]
I believe this has been addressed on master already (and tests added [here](https://github.com/rwjblue/ember-template-lint/blob/a48191c1e3820f7cf304eba15af1cbcf87dfeeef/test/unit/rules/lint-block-indentation-test.js#L146-L152)).

Closing, but please open an issue if I missed something...

**Pull request closed without merging**